docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md

```diff
@@ -169,16 +182,15 @@ The key you provide should look like:
 
 Grafana supports user authentication through Azure AD, which is useful when you want users to access Grafana using single sign-on. This topic shows you how to configure SAML authentication in Grafana with [Azure AD](https://azure.microsoft.com/en-us/services/active-directory/).
 
-**Before you begin:**
+**Before you begin**
+
+Ensure you have permission to administer SAML authentication. For more information about roles and permissions in Grafana, refer to [Roles and permissions](/docs/grafana/<GRAFANA_VERSION>/administration/roles-and-permissions/).
+
+Learn the [limitations of Azure AD SAML] (https://learn.microsoft.com/en-us/entra/identity-platform/id-token-claims-reference#groups-overage-claim) integration.
+
+Configure SAML integration with Azure AD, [creating an app integration](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/add-application-portal-configure) inside the Azure AD organization first.
```

Comment:
I guess the link included here doesn't cover the steps for creating the integration.
```suggestion
Configure SAML integration with Azure AD, [creating an Enterprise Application](https://learn.microsoft.com/en-us/entra/identity/enterprise-apps/add-application-portal) inside the Azure AD organization first and then [enable single sign-on](https://learn.microsoft.com/en-us/entra/identity/enterprise-apps/add-application-portal-setup-sso).
```